src/app/CommandHandler.cpp

```diff
@@ -64,7 +64,20 @@ CHIP_ERROR CommandHandler::AllocateBuffer()
         mCommandMessageWriter.Init(std::move(commandPacket));
         ReturnErrorOnFailure(mInvokeResponseBuilder.InitWithEndBufferReserved(&mCommandMessageWriter));
 
-        mInvokeResponseBuilder.SuppressResponse(mSuppressResponse);
+        if (mReserveSpaceForMoreChunkMessages)
+        {
+            ReturnErrorOnFailure(mInvokeResponseBuilder.ReserveSpaceForMoreChunkedMessages());
+        }
+
+        // Sending a response command to a response command is going to be removed from spec soon.
```

Comment:
"Response command to a response command" ? Not sure what that is.

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/data_model/Interaction-Model.adoc?plain=1#L1289

`InvokeResponse` to an `InvokeResponse`. It will be removed from spec soon, and was never used